newdle/client/src/components/Navigator.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Container, Header} from 'semantic-ui-react';
+
+import styles from './Navigator.module.scss';
+
+export default function Navigator({steps, active}) {
+  return (
+    <Container>
+      {steps.map((step, index) => (
+        <Step key={index} active={index + 1 === active}>
+          {index + 1}
+        </Step>
+      ))}
+      {steps.map((step, index) => {
```

Comment:
This whole block can be replaced with `<StepBody title={steps[active-1].title} description={steps[active-1].description} />`

Or, to make it prettier, assign `const activeStep = steps[active - 1];` at the beginning of the function and then use `activeStep` in the snippet above